nested/track.json

```diff
@@ -0,0 +1,52 @@
+{
+	"short-description": "Nested query benchmark using up to 11,203,029 questions from StackOverflow",
+	"description": "This test indexes and queries up to 11,203,0299 questions from StackOverflow (total 3.64 GB json)",
+	"data-url": "http://benchmarks.elasticsearch.org.s3.amazonaws.com/corpora/nested",
+	"indices": [{
+		"name": "sonested",
+		"types": [{
+			"name": "question",
+			"mapping": "mappings.json",
+			"documents": "documents.json.bz2",
+			"document-count": 11203029,
+			"compressed-bytes": 695293381,
+			"uncompressed-bytes": 3637747670
+		}]
+	}],
+	"operations": [{
+		"name": "index-append",
+		"operation-type": "index",
+		"bulk-size": 500
+	}, {
+		"name": "force-merge",
+		"operation-type": "force-merge"
+	},  {
+		"name": "randomized-nested-queries",
+		"operation-type": "search",
+		"param-source": "nested-query-source",
+		"cache": false
+	}],
+	"challenges": [{
+		"name": "nested-search-challenge",
+		"description": "Indexes document corpus for an hour using Elasticsearch default settings . After that randomized nested queries are run.",
+		"default": true,
+		"index-settings": {
+			"index.number_of_replicas": 0
+		},
+		"schedule": [{
+			"operation": "index-append",
+			"warmup-time-period": 20,
```

Comment:
I would increase the warmup time to 120 or maybe 240 seconds. You see that 20 seconds is too small because of the high variation between minimum and median indexing rate in the summary report.